src/cli/bootupctl.rs

```diff
@@ -58,6 +58,8 @@ pub enum CtlVerb {
     AdoptAndUpdate,
     #[clap(name = "validate", about = "Validate system state")]
     Validate,
+    #[clap(name = "migrate", about = "Migrate a system to static a GRUB config")]
```

Comment:
Migrate is a bit generic, should this be migrate-grub ?
```suggestion
    #[clap(name = "migrate-grub", about = "Migrate a system to a static GRUB config")]
```

Reply:
Sure, I'll do that.